change/twoSoundManager

rnjsrbals opened the pull request.

rnjsrbals committed:
11/07 수정
2인용 플레이시 한명이 게임 오버시 나머지 한명이 플레이할 때 브금 중단 현상 해결

rnjsrbals committed:
11/07 수정
2인용 플레이시 한명이 게임 오버시 나머지 한명이 플레이할 때 브금 중단 현상 해결
11/08 수정
수정(추가) 사항 : TwoPlayerScreen.java  => checkLevelCompletion, update, updatePlayerStatus, run, runGameLoop

rnjsrbals committed:
11/07 수정
2인용 플레이시 한명이 게임 오버시 나머지 한명이 플레이할 때 브금 중단 현상 해결
11/08 수정
2인용 플레이에서 한명이 먼저 해당 레벨 완료(게임 오버 or 클리어)시 다른 플레이어가 게임 완료될 때까지 멈추도록 설정 => 나머지 플레이어가 해당 레벨 완료(게임 오버 or 클리어)시 플레이어 모두가 동시에 다음 레벨로 가도록 설정

yeyoungyun commented on src/engine/SoundManager.java:
```diff
@@ -87,6 +89,13 @@ private SoundManager() {
             loadSound(Sound.BGM_LV5, "res/sound/BGM/Lv5.wav");
             loadSound(Sound.BGM_LV6, "res/sound/BGM/Lv6.wav");
             loadSound(Sound.BGM_LV7, "res/sound/BGM/Lv7.wav");
+            loadSound(Sound.BGM, "res/sound/BGM/Lv1.wav");
```
"BGM" 이름의 사운드를 새로 load하지말고 기존에 있는 "BGM_LV1"을 이용하는 것이 좋을 것 같습니다. 그 아래의 주석도 불필요하므로 제거 부탁드립니다.

rnjsrbals committed:
11/07 수정
2인용 플레이시 한명이 게임 오버시 나머지 한명이 플레이할 때 브금 중단 현상 해결
11/08 수정
2인용 플레이에서 한명이 먼저 해당 레벨 완료(게임 오버 or 클리어)시 다른 플레이어가 게임 완료될 때까지 멈추도록 설정 => 나머지 플레이어가 해당 레벨 완료(게임 오버 or 클리어)시 플레이어 모두가 동시에 다음 레벨로 가도록 설정

rnjsrbals committed:
11/12 수정

rnjsrbals committed:
11/17 수정
